lib/galaxy/datatypes/util/gff_util.py

```diff
@@ -572,3 +572,18 @@ def key_fn(fields):
     for chrom_features in chroms_features_sorted:
         for feature in chrom_features:
             yield feature
+
+
+__all__ = (
+    "GenomicInterval",
```

Comment:
Can we not export this and fix `lib/galaxy/datatypes/converters/gff_to_interval_index_converter.py` to import it from `bx.intervals.io` (it's already import from bx-python any way).

Reply:
The counter point would be that once you explicitly add the re-exports to `__all__`, they become part of the module interface and it becomes harder to remove them.